assets/src/js/header.js

```diff
@@ -227,5 +227,44 @@ export const setupHeader = () => {
     }
   });
 
+  /**
+   * Set the submenu behavior.
+   * Desktop should render drop/down list
+   * Mobile should render an accordion list
+   */
+  const setupSubmenu = () => {
+    document.querySelectorAll('.nav-submenu').forEach(submenu => {
+      // Find its parent menu item
+      const menuItem = document.querySelector(`*[data-menu="${submenu.dataset.parentMenu}"]`);
+      const list = submenu.querySelector('ul');
+
+      // The limits should be 6 for a scrollable list
+      if(list.children.length >= 6) {
+        list.classList.add('scrollable');
+      }
+
+      submenu.addEventListener('mouseover', evt => {
```

Comment:
I think this could be achieved in CSS directly with the `:hover` pseudo-element on the `li` elements, by changing the `display` value of their corresponding dropdown menus